plottable.d.ts

```diff
@@ -3578,7 +3578,7 @@ declare module Plottable {
 
 declare module Plottable {
     module Dispatchers {
-        type KeyCallback = (keyCode: number, event: KeyboardEvent) => void;
```

Comment:
Yes, but the `KeyInteraction` does not utilize this `KeyCallback`.  It still is not clear to me why we need optional parameters here.